src/rockstor/scripts/scheduled_tasks/snapshot.py

```diff
@@ -63,7 +68,7 @@ def delete(aw, share, snap_type, prefix, max_count):
         name__startswith=prefix).order_by('-id')
     for snap in snapshots[max_count:]:
         try:
-            url = ('shares/%s/snapshots/%s' % (share.name, snap.name))
+            url = ('shares/%s/snapshots/%s' % (share.id, snap.name))
```

Comment:
since share.id is an integer,  it should be %d. But even better to just use `'shares/{}/snapshots/{}'.format(share.id, snap.name)` so we don't have to deal with types ourselves.

Reply:
Agreed, I had thought we would run into the same unicode issue as in #1776 with %d use but as you say in this context that is not the case. However as advised I have move to str.format for the url string formats at least; much nicer.